linkerd/conf.yaml.example

```diff
@@ -0,0 +1,36 @@
+init_config:
+
+instances:
+
+    # The linkerd prometheus endpoint
+    #
+  - prometheus_url: http://localhost:9990/admin/metrics/prometheus
+
+    # Set (optional) name of the instance. Will appear as tag "linkerd_instance"
+    # on datadog
+    #
+    # name: "linkerd"
```

Comment:
Let's remove this and let the user decide how to do it https://github.com/DataDog/integrations-core/blob/28ec869b3854e1df533e6240bd22501494097c5a/nginx/conf.yaml.example#L19